datagenerator/src/main/resources/rythm/enum.rythm

```diff
@@ -0,0 +1,21 @@
+@import com.linkedin.pegasus.generator.spec.ClassTemplateSpec
+@args String className
+@args List<String> symbols
+@args Map<String, String> symbolDocs
+
+// DO NOT EDIT - THIS FILE IS GENERATED BY restli-csharp
+
+/// <summary>
```

Comment:
We should only generate class comment if the enum pdsc has "doc" specified. Here no need to put <summary> and can directly put that "doc" value.